routers/user/home.go

```diff
@@ -74,6 +74,7 @@ func retrieveFeeds(ctx *context.Context, user *models.User, includePrivate, isPr
 	if ctx.User != nil {
 		userCache[ctx.User.ID] = ctx.User
 	}
+	commentCache := map[int64]*models.Comment{}
```

Comment:
As far as I can tell, there will never be two actions with the same (non-zero) `CommentID`, so I don't see any point in caching comments.